versioned_docs/version-4/studio-community-plan.mdx

```diff
@@ -19,5 +19,6 @@ Our Community plan features include the following:
 - Unlimited [public projects](projects.mdx#change-a-projects-visibility)
 - [Design and simulate flows](studio.mdx#stately-studios-editor)
 - [Import from code](import-from-code.mdx)
-- [Export as code](export-as-code.mdx)
+- [Export code as flows](export-as-code.mdx)
```

Comment:
Not sure what “export code as flows” means?

Reply:
Ha! That's supposed to be "Export flows as code".

Reply:
Fixed!